deployment/app.py

```diff
@@ -0,0 +1,476 @@
+                            img_io = StringIO()
+                            images[i].save(img_io, 'JPEG', quality=70)
+                            img_io.seek(0)
+                            zf.writestr(str(i) + '.jpg', img_io.getvalue())
+                    return send_file(mf, mimetype='application/zip')
+
+        elif info["type"] ==  ModelType.activity_classification or \
+             info["type"] == ModelType.text_classification or \
+             info["type"] == ModelType.random_forest_classifier or \
+             info["type"] == ModelType.decision_tree_classifier or \
+             info["type"] == Modeltype.boosted_tree_classifier or \
+             info["type"] == ModelType.nearest_neighbor_classifier or \
+             info["type"] == ModelType.svm_classifier or \
+             info["type"] == ModelType.random_forest_regression or \
+             info["type"] == ModelType.decision_tree_regression or \
+             info["type"] == ModelType.linear_regression or \
+             info["type"] == ModelType.boosted_tree_regression or \
+             info["type"] == ModelType.kmeans or \
+             info["type"] == ModelType.item_similarity_recommender or \
+             info["type"] == ModelType.item_content_recommender or \
+             info["type"] == ModelType.popularity_recommender or \
+             info["type"] == ModelType.factorization_recommender:
+
+            loaded_data = request.get_json()
+            data = json.dumps(loaded_data)
+            loaded_data = json.loads(data)
+            
+            if not all(x in info["features"] for x in loaded_data.keys):
+                return return_success({"error" : "Column needed for inference is missing or \
+                    the column name given does not match the model info"}, False)
+
+            inp = {}
+            for key in loaded_data.keys():
+                inp[key]=[loaded_data[key]]
+            sf_data = tc.SFrame(inp)
+            prediction =  model.predict(sf_data)
+            if info["type"] == ModelType.random_forest_regression or \
+             info["type"] == ModelType.decision_tree_regression or \
+             info["type"] == ModelType.linear_regression or \
+             info["type"] == ModelType.boosted_tree_regression:
+                return return_success({"Estimated Value" : str(prediction[0])})
+            if info["type"] == ModelType.item_similarity_recommender or \
+             info["type"] == ModelType.item_content_recommender or \
+             info["type"] == ModelType.popularity_recommender or \
+             info["type"] == ModelType.factorization_recommender:
+                return return_success({"Recommendation" : str(prediction[0])})
+            if info["type"] == ModelType.kmeans:
+                return return_success({"Cluster" : str(prediction[0])})
+            return return_success({"Label": str(prediction[0])}, True)
+
+        elif info["type"] == ModelType.sound_classification:
+            # TODO 
```

Comment:
You could let it fall into the else case below for now (an "Unimplemented Toolkit" error seems about as good a TODO as any).